### PR TITLE
Eliminate ErrorProne warnings

### PR DESCRIPTION
**Pull request by rainyShine**
For #3359.

### CODE REVIEW

**Comment by rainyShine** [comment]
@googlebot I signed it!

**Comment by chanseokoh** [comment]
@rainyShine thanks for the PR. The builds are failing due to style violation. Try running `./gradlew googleJavaFormat`.

**Comment by rainyShine** [comment]
@chanseokoh Certainly.
I'm sorry to reread the guidelines.And I try that.

**Comment by rainyShine** [comment]
@chanseokoh I've run below command `./gradlew googleJavaFormat` and pushed.
Could you check again?

File: jib-cli/src/main/java/com/google/cloud/tools/jib/cli/buildfile/BaseImageSpec.java [hunk 1]
```diff
@@ -29,8 +29,8 @@
  * <pre>{@code
  * image: gcr.io/example/baseimage
  * platforms:
- *   - see {@link PlatformSpec}
- *   - see {@link PlatformSpec}
+ *   - see }{@link PlatformSpec}{@code
+ *   - see }{@link PlatformSpec}{@code
```
Comment (suztomo) [inline jib-cli/src/main/java/com/google/cloud/tools/jib/cli/buildfile/BaseImageSpec.java:33]: This Javadoc looks strange to me. Which error prone warnings is this addressing?

**Comment by chanseokoh** [comment]
This is for #3359.

File: jib-cli/src/main/java/com/google/cloud/tools/jib/cli/buildfile/BaseImageSpec.java (hunk 1 above)
Comment (rainyShine) [reply jib-cli/src/main/java/com/google/cloud/tools/jib/cli/buildfile/BaseImageSpec.java:33]: A warning is given where another javadoc tag(i.e. @link) is included inside @code.
I referred to the link below.  
https://stackoverflow.com/questions/8208032/is-nesting-of-javadoc-tag-codelink-allowed

Comment (suztomo) [reply jib-cli/src/main/java/com/google/cloud/tools/jib/cli/buildfile/BaseImageSpec.java:33]: Got it. Thanks.

Comment (chanseokoh) [reply jib-cli/src/main/java/com/google/cloud/tools/jib/cli/buildfile/BaseImageSpec.java:33]: Interesting. Looks good then.

**Review by chanseokoh — APPROVED** [review]
@rainyShine thanks a lot of fixing these! Merging now.

**Pull request merged**